src/drawing.rs

```diff
@@ -236,9 +254,31 @@ impl Drawing {
     ///         // Rotate to the right (clockwise) by 1 degree
     ///         turtle.right(1.0);
     ///     }
-    ///
+    ///     # #[cfg(feature = "docs_image")]
+    ///     # turtle_docs_helper::save_docs_image(&drawing, "circle");
+    /// # }
+    /// ```
+    /// ```rust, no_run
+    /// # use turtle::Drawing;
+    ///     # let mut drawing = Drawing::new();
+    ///     # let mut turtle = drawing.add_turtle();
     ///     turtle.wait_for_click();
+    /// ```
+    /// ```rust
+    /// # use turtle::Drawing;
+    /// # fn main() {
+    ///     # let mut drawing = Drawing::new();
+    ///     # let mut turtle = drawing.add_turtle();
+    ///
+    ///     # for _ in 0..360 {
+    ///     #    // Move forward three steps
+    ///     #    turtle.forward(3.0);
+    ///     #    // Rotate to the right (clockwise) by 1 degree
+    ///     #    turtle.right(1.0);
+    ///     # }
     ///     drawing.set_center([50.0, 100.0]);
+    ///     # #[cfg(feature = "docs_image")]
+    ///     # turtle_docs_helper::save_docs_image(&drawing, "circle_offset_center");
```

Comment:
As orignally the code was `no_run` I have gone with using:
```
    ///     # #[cfg(doctests_run_user_input)]
    ///     turtle.wait_for_click();
```
As the flag `doctests_run_user_input` should never be defined therefore the code should never run. And it is completely independant from `docs_images` so that being the docs with or without the images flag will work. 

Otherwise with `# #[cfg(not(docs_image))]` it would just poll forever for user input if a user ran `cargo doc` without the flag set.